polus-image-quality-plugin/src/main.py

```diff
@@ -0,0 +1,87 @@
+from bfio import BioReader
```

Comment:
All files 
----------
1) I'm not sure how much we care of pep8
2) No license boilerplate in modules.
3) Numerous smileys in docstrings and in the readme text.

File dom.py:
------------
line 6 -- missing class docstring DOM. See 3.8.4 of Google's python style guide https://google.github.io/styleguide/pyguide.html for further details 

File main.py:
-------------
line 2 -- multiple imports -- "import argparse, logging, os"
lines 19, 29 -- messy comments

Utils.py:
---------
line 5 -- multiple imports
line 128 -- whitespaces out of order
lines 192-208 -- blank lines
line 61 -- method local_Focus_score() is not documented
line 75 -- method saturation_calculation() is undocumented
line 92 -- mathod  brisque_calculation() is undocumented
line 98 -- docstring of method  sharpness_calculation() -- smiley
line 110 -- undocumented calculate_correlation_dissimilarity()
line 135 -- undocumented rps()
line 160 -- undocumented power_spectrum()
line 186 -- undocumented  write_csv()

Tests:
-------
Texture metrics (correlation, dissimilarity) seem not covered by a test